app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChatroomsController < ApplicationController
+  
```

Comment:
This controller's action is available to all type of users, right? I meant about the authentication and the authorization. :D